dags/openshift_nightlies/util/executor.py

```diff
@@ -23,6 +23,10 @@ def get_default_executor_config(dag_config: DagConfig, executor_image='airflow-a
         }
 
 def get_executor_config_with_cluster_access(dag_config: DagConfig, release: OpenshiftRelease, executor_image="airflow-ansible"):
+    if release.platform == "aro":
+        return ""
```

Comment:
@whitleykeith this problem for aro has been fixed. Told david to use the same logic that's used for prebuilt by generating a kubeconfig and using that while generating the secrets.